src/Microsoft.Data.SqlClient/tests/FunctionalTests/SqlExceptionTest.cs

```diff
@@ -33,9 +34,9 @@ public void SerializationTest()
             Assert.Equal(e.StackTrace, sqlEx.StackTrace);
         }
 
-
         [Fact]
-        [ActiveIssue("12161", TestPlatforms.AnyUnix)]
+        [ActiveIssue("12161", TargetFrameworkMonikers.Netcoreapp)]
```

Comment:
I see there's another PR to remove this test, please revert these changes from this PR to concentrate of NetFx driver changes for BinaryFormatter only.